demos/general/demo_pipeline_nwb.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
```

Comment:
This demo does not add much compared to other demos. Better modify existing demos to include a save function where NWB is an option.